core/src/main/scala/com/velocidi/apso/Implicits.scala

```diff
@@ -89,19 +98,20 @@ object Implicits {
   }
 
   /** Implicit class that provides new methods for maps.
-    * @param map the map to which the new methods are provided.
+    * @param map
+    *   the map to which the new methods are provided.
     */
   final implicit class ApsoMap[A, B](val map: Map[A, B]) extends AnyVal {
 
-    /** Merges a given map with this map. The map is constructed as follows:
-      * <ul>
-      *   <li>Keys present in one of thw two maps are present in the merged map;
-      *   <li>Keys in both maps are present in the merged map with a value given
-      *       by `f(thisValue, thatValue)`;
-      * </ul>
-      * @param that the map to be merged with this map
-      * @param f the function used to merge two values with the same key
-      * @return the merged map.
+    /** Merges a given map with this map. The map is constructed as follows: <ul> <li>Keys present in one of thw two
+      * maps are present in the merged map; <li>Keys in both maps are present in the merged map with a value given by
+      * `f(thisValue, thatValue)`; </ul>
```

Comment:
Can we avoid HTML so that this doesn't get reformatted?

```suggestion
    /** Merges a given map with this map. The map is constructed as follows:
      *   - Keys present in one of thw two maps are present in the merged map;
      *   - Keys in both maps are present in the merged map with a value given by `f(thisValue, thatValue)`;
      *
```